Formula/q/qt@5.rb

```diff
@@ -180,42 +179,6 @@ class QtAT5 < Formula
     directory "qtbase"
   end
 
-  # CVE-2023-32573
-  # Original (malformed with CRLF): https://download.qt.io/official_releases/qt/5.15/CVE-2023-32573-qtsvg-5.15.diff
-  # Remove with Qt 5.15.14
-  patch do
-    url "https://invent.kde.org/qt/qt/qtsvg/-/commit/5b1b4a99d6bc98c42a11b7a3f6c9f0b0f9e56f34.diff"
-    sha256 "0a978cac9954a557dde7f0c01e059a227f2e064fe6542defd78f37a9f7dd7a3d"
-    directory "qtsvg"
-  end
-
-  # CVE-2023-32762
-  # Original (malformed with CRLF): https://download.qt.io/official_releases/qt/5.15/CVE-2023-32762-qtbase-5.15.diff
-  # Remove with Qt 5.15.14
-  patch do
-    url "https://invent.kde.org/qt/qt/qtbase/-/commit/1286cab2c0e8ae93749a71dcfd61936533a2ec50.diff"
-    sha256 "2fba1152067c60756162b7ad7a2570d55c9293dd4a53395197fd31ab770977d7"
-    directory "qtbase"
-  end
-
-  # CVE-2023-32763
-  # Original (malformed with CRLF): https://download.qt.io/official_releases/qt/5.15/CVE-2023-32763-qtbase-5.15.diff
-  # Remove with Qt 5.15.15
```

Comment:
This one might still apply because it says 5.15.15.

Reply:
sounds about right.